File: M2D/Traductor-clipspy/output.py
class Clase2:
   def __init__(self):
      self.Numero=None
      self.Direccion=None
   def getNumero(self):
      return self.Numero
   def setNumero(self,Numero):
      self.Numero = Numero
   def getDireccion(self):
      return self.Direccion
   def setDireccion(self,Direccion):
      self.Direccion = Direccion

class Clase1:
   def __init__(self):
      self.Nombre=None
      self.clase2List1=set()
   def getNombre(self):
      return self.Nombre
   def setNombre(self,Nombre):
      self.Nombre = Nombre
   def getClase2List1(self):
      return self.clase2List1
   def setClase2List1(self,clase2List1):
      self.clase2List1 = clase2List1
   def Añadir(self):
      pass

File: M2D/Traductor-clipspy/output.java

// Java code for class Clase2
public class Clase2 {
   public int Numero;
   public int getNumero() {
      return this.Numero;
   }
   public void setNumero(int Numero) {
      this.Numero = Numero;
   }
   public String Direccion;
   public String getDireccion() {
      return this.Direccion;
   }
   public void setDireccion(String Direccion) {
      this.Direccion = Direccion;
   }
}

// Java code for class Clase1
public class Clase1 {
   public String Nombre;
   public String getNombre() {
      return this.Nombre;
   }
   public void setNombre(String Nombre) {
      this.Nombre = Nombre;
   }
   private HashSet<Clase2> clase2List1;
   public HashSet<Clase2> getClase2List1() {
      return this.clase2List1;
   }
   public void setClase2List1(HashSet<Clase2> clase2List1) {
      this.clase2List1 = clase2List1;
   }
   public void Añadir() {
      // method body
   }
}

